odoo/addons/base/views/res_users_identitycheck_views.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="identity_check_wizard" model="ir.ui.view">
+            <field name="model">res.users.identitycheck</field>
+            <field name="arch" type="xml">
+                <form string="Password Confirmation">
+                    <sheet class="bg-primary">
+                        <h3><strong>Please enter your password to confirm you own this account</strong></h3>
+                        <br/>
```

Comment:
Is it necessary to have a br between h3 and a div ?

Reply:
This entire record is something I've moved from res_users_views.xml to res_users_identitycheck_views.xml. Checking the others views I put in this file was not really in the scope of my PR so I didn't want to edit it. Though, if you think it would be beneficial to take advantage of moving code to improve it, I can do that. Should I ? @tde-banana-odoo